docs/quickstarts/1_client_credentials.rst

```diff
@@ -93,6 +93,17 @@ Loading the resource and client definitions happens in ``Startup.cs`` - the temp
             .AddInMemoryIdentityResources(Config.GetIdentityResources())
             .AddInMemoryApiResources(Config.GetApis())
             .AddInMemoryClients(Config.GetClients());
+        
+        // To know how to obtain the Environment property,
+        // take a look at the full Startup.cs file in our reporisory
+        if (Environment.IsDevelopment())
+        {
+            builder.AddDeveloperSigningCredential();
+        }
+        else
+        {
+            throw new Exception("need to configure key material");
```

Comment:
I have issues with telling people to add throw an exception to their code when its not run in development. 

That being said most of our samples are just showing how to run things in development (Set to run http not https)  we could just remove the else.

Reply:
All right, I've just removed it in the next [commit](https://github.com/IdentityServer/IdentityServer4/pull/3273/commits/8056411d224c85de02b091a9499f80b0f3e087d4).
By the way, I took inspiration fro the else statement from your QuickStart examples [here](https://github.com/IdentityServer/IdentityServer4/blob/master/samples/Quickstarts/1_ClientCredentials/src/IdentityServer/Startup.cs)

Reply:
Which means that I should probably add an issue to remove the else from the sample.

Reply:
It'll throw an exception anyway, yeah? One that is in IdentityServer, rather than the implementors code.

Reply:
Thats a good question.  I would hope that it threw an exception anyway.